lib-multisrc/madara/src/eu/kanade/tachiyomi/multisrc/madara/Madara.kt

```diff
@@ -723,7 +723,7 @@ abstract class Madara(
     }
 
     // Manga Details Selector
-    open val mangaDetailsSelectorTitle = "div.post-title h3, div.post-title h1, #manga-title > h1"
+    open val mangaDetailsSelectorTitle = "div.post-title h3, div.post-title h1, #manga-title > h1, div.summary_content h2"
```

Comment:
do you know if any other source uses this selector as well? if not then just override the specific source.

Reply:
Ok

Reply:
I searched through a few sources, but didn't find another matching manga details title identifier